docs/docs-content/palette-cli/commands/validator.md

```diff
@@ -140,6 +153,13 @@ validatorconfigs.validation.spectrocloud.labs    2023-11-09T21:02:12Z
 You can use the `kubectl` command to view the validation results. To review all the results collectively, use the
 `describe` command to display the `validationresults` CR.
 
+:::tip
+
+Use the `validator describe` command to view the validation results. The `validator describe` provides a more
```

Comment:
```suggestion
Use the `validator describe` command to view the validation results. `validator describe` provides a more
```

Reply:
Or _The `validator describe` command_ :)

Reply:
Ooo let's go with the later suggestion 😄